internal/server/api.yaml

```diff
@@ -486,7 +489,6 @@ components:
           example: 'subscription.rhsm.redhat.com'
         base-url:
           type: string
-          format: url
```

Comment:
ah was this erroring?

Reply:
yes, I think it could be replaced with uri but wasn't sure at the time if they were interchangeable